### PR TITLE
Improve code and tests for blog feed fetching/parsing

### PR DESCRIPTION
**Pull request by pmac**
Refs Bug 1303794

### CODE REVIEW

**Comment by pmac** [comment]
@jgmize r?

File: bedrock/mozorg/models.py [hunk 1]
```diff
@@ -129,3 +131,42 @@ def blog_title(self):
     @property
     def blog_link(self):
         return settings.BLOG_FEEDS[self.blog_slug]['url']
+
+    @classmethod
+    def parse_feed(cls, feed_id, feed_options):
```
Comment (jgmize) [inline bedrock/mozorg/models.py]: NIT: this `classmethod` doesn't actually use `cls` and could just be a regular function instead.

Comment (pmac) [inline bedrock/mozorg/models.py]: They all could. I just couldn't find a better place for them, and adding them to the model class made imports cleaner. I could move them to a utils.py, but I think that the models file imports from utils, and so I can't import the model there w/o some inline thing... I'm conflicted on the best location, but this seems okay. I can move it again tho.

Comment (jgmize) [inline bedrock/mozorg/models.py]: Fair enough. I don't want to nitpick this any further; this is still an improvement over the current code and I'd rather focus our efforts on other things. As always, NITs can be safely ignored and aren't blockers for merge.

**Pull request merged**